.github/workflows/cypress_workflow.yml

```diff
@@ -42,18 +40,65 @@ env:
   COMMENT_TAG: '[MANUAL CYPRESS TEST RUN RESULTS]'
   COMMENT_SUCCESS_MSG: ':white_check_mark: Cypress test run succeeded!'
   COMMENT_FAILURE_MSG: ':x: Cypress test run failed!'
+  LATEST_VERSION: '2.17.0'
 
 jobs:
   cypress-tests:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-          group: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+        include:
+          # Standard test configs (groups 1-9)
+          - group: 1
+            config: standard
+            test_location: ftr
+          - group: 2
+            config: standard
+            test_location: ftr
+          - group: 3
+            config: standard
+            test_location: ftr
```

Comment:
we have a variable FTR_PATH, should we use this/change FTR_PATH to something that makes more sense to usage and add the other locations? or should we get rid of the FTR_PATH variable